ext/uri/php_uriparser.h

```diff
@@ -28,8 +28,6 @@ typedef struct uriparser_uris_t {
 	bool normalized_uri_initialized;
 } uriparser_uris_t;
 
-PHP_MINIT_FUNCTION(uri_uriparser);
```

Comment:
I got rid of this function and I inlined it into php_uri the same way how lexbor does it. This is not entirely a separate change because I added `PHPAPI` to `uri_handler_register ` that have been forgotten to be done before, therefore I moved the function to `php_uri.h` - which is not used by `php_uriparser.c` (and I didn't want to fiddle with `extern`).